File: DemoProject/src/main/java/com/core/SelectionSort.java
package com.core;

public class SelectionSort {

	public static void main(String[] args) {
		int[] num = { 4, 6, 2, 1, 8, 7 };
		int small = 0, swap = 0, i, j;
		for (i = 0; i < num.length - 1; i++)
			for (j = i + 1; j < num.length; j++) {
				if (num[i] > num[j]) {
					// i=j;
					swap = num[i];
					num[i] = num[j];
					num[j] = swap;
				}
			}
		System.out.println("array elements are "+num.toString());
		for (i = 0; i < num.length; i++) {
			// for( j=i-1;j>i;j--) {
			System.out.print(num[i]+" ");
		}
	}

}

File: nodedemos/selectionsort.js
let elements = [5, 8, 2, 9, 3, 1, 4, 6];
console.log("The array elements before sorting are " + elements);
for (var i = 0; i < elements.length - 1; i++) {
    for (var j = i + 1; j < elements.length; j++) {
        if (elements[i] > elements[j]) {
            var temp = elements[i];
            elements[i] = elements[j];
            elements[j] = temp;
        }
    }
}
console.log("The array elements are::" + elements);